fieldservice/models/res_config_settings.py

```diff
@@ -63,6 +66,15 @@ class ResConfigSettings(models.TransientModel):
         related='company_id.auto_populate_persons_on_location',
         readonly=False)
 
+    @api.onchange('group_fsm_equipment')
+    def _onchange_group_fsm_equipment(self):
+        self.auto_populate_the_equipments = self.group_fsm_equipment
```

Comment:
@murtuzasaleh 

If `group_fsm_equipment` is true, `auto_populate_the_equipments` can still be true or false.

Reply:
@max3903 Improved code, please check it again.